api/soap/mantisconnect.wsdl

```diff
@@ -546,6 +550,16 @@
   <part name="issue" type="tns:IssueData" /></message>
 <message name="mc_issue_addResponse">
   <part name="return" type="xsd:integer" /></message>
+<message name="mc_issue_reportRequest">
+  <part name="username" type="xsd:string" />
+  <part name="password" type="xsd:string" />
+  <part name="issue" type="tns:IssueData" />
+  <part name="attach_note" type="xsd:string" />
+  <part name="file_name" type="xsd:string" />
+  <part name="file_type" type="xsd:string" />
+  <part name="content" type="xsd:base64Binary" /></message>
+<message name="mc_issue_reportResponse">
+  <part name="return" type="tns:IssueData" /></message>
```

Comment:
That would be IssueReportData or something along these lines containing IssueData as an element.